doorkeeper-orms.gemspec

```diff
@@ -10,17 +10,23 @@ Gem::Specification.new do |s|
   s.authors     = ["jasl"]
   s.email       = ["jasl9187@hotmail.com"]
   s.homepage    = "http://github.com/doorkeeper-gem/doorkeeper-orms"
-  s.summary     = "Doorkeeper with extracted ORM specifics, including mongoid 2-4 and mongo_mapper."
-  s.description = "Doorkeeper with extracted ORM specifics, including mongoid 2-4 and mongo_mapper"
+  s.summary     = "Doorkeeper mongoid 2, 3, 4 and mongo_mapper ORMs"
+  s.description = "Doorkeeper mongoid 2, 3, 4 and mongo_mapper ORMs"
```

Comment:
Is the plan to have one repo for each ORM? If so, how will we make sure each repo can be tested against the same Doorkeeper tests/specs?